src/erlcloud_ec2.erl

```diff
@@ -185,7 +190,8 @@
 % -define(NEW_API_VERSION, "2013-10-15").
 % -define(NEW_API_VERSION, "2014-02-01").
 % -define(NEW_API_VERSION, "2014-06-15").
--define(NEW_API_VERSION, "2015-10-01").
+% -define(NEW_API_VERSION, "2015-10-01").
```

Comment:
I was just trying to follow the existing style. If you prefer to remove older version api, I am happy to do that.